cpp/example/example.cc

```diff
@@ -27,7 +29,26 @@ class Counter {
   }
 };
 
-int main() {
+DEFINE_string(redis_address, "", "The ip address of redis server.");
+
+DEFINE_string(dynamic_library_path, "", "The local path of the dynamic library.");
+
+int main(int argc, char **argv) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
+  const std::string dynamic_library_path = FLAGS_dynamic_library_path;
+  const std::string redis_address = FLAGS_redis_address;
+  gflags::ShutDownCommandLineFlags();
+
+  RAY_CHECK(!dynamic_library_path.empty()) << "Please add a local dynamic library by '--dynamic-library-path'";
+
+  ray::api::RayConfig::GetInstance()->lib_name = dynamic_library_path;
+  if (!redis_address.empty()) {
+    ray::api::RayConfig::GetInstance()->SetRedisAddress(redis_address);
+  }
+
+  /// initialization to cluster mode
+  ::ray::api::RayConfig::GetInstance()->run_mode = RunMode::CLUSTER;
+
   /// initialization
   Ray::Init();
 
```

Comment:
Can we put here cpp/src/ray/example/example.cc instead of cpp/src/example/example.cc?

Reply:
I think the example should be a independent folder because it can be used breaking away from ray project. It shouldn't be located in `src` folder. You can see some other project.
https://github.com/gabime/spdlog/tree/v1.x/example
https://github.com/msgpack/msgpack-c/tree/cpp_master/example

Reply:
What I 

I agree that we should put it in cpp/example/example.cc but what I meant is that the example in cpp/src/_ray_/example/example.cc is better than the example in cpp/src/example/example.cc, can we use  cpp/src/_ray_/example/example.cc?

Reply:
Got it.